.vscode/launch.json

```diff
@@ -1,6 +1,24 @@
 {
   "version": "0.2.0",
   "configurations": [
+    {
+      "type": "node",
+      "name": "Frontend Debug",
+      "request": "launch",
+      "cwd": "${workspaceFolder}/grassroots-frontend",
+      "program": "${workspaceFolder}/grassroots-frontend/node_modules/.bin/vite",
+      "args": ["--host"],
+      "console": "integratedTerminal"
+    },
+    {
+      "type": "node",
+      "name": "Backend Debug",
+      "request": "launch",
+      "cwd": "${workspaceFolder}/grassroots-backend",
+      "program": "${workspaceFolder}/grassroots-backend/node_modules/.bin/nest",
+      "args": ["start", "--watch", "--debug"],
```

Comment:
Should this run build and execute from dist?

Reply:
based on convo in slack we'll leave this as is

TL;DR: using watch + debug doesn't work in chorme, but does work in VSCode